packages/themes/scss/bright-blue.scss

```diff
@@ -0,0 +1,59 @@
+// Commented out because the original css didn't import
```

Comment:
Let's go ahead and use Lato, it doesn't matter if the student implementation had a bug and wasn't really using it.

Reply:
Resolved!